tests/parsing/GlobbingTest.py

```diff
@@ -222,8 +222,9 @@ class GlobTest(unittest.TestCase):
     def setUp(self):
         self.maxDiff = None
 
-    def _test_glob(self, pattern, file_list):
-        results = sorted([os.path.normcase(g) for g in glob(pattern)])
+    def _test_glob(self, pattern, file_list, ignored_globs=None):
```

Comment:
aaaah nvm... why the heck is there just an underscore before the function name, I thought it was a test... not your fault :3